src/BlobStore.hs

```diff
@@ -161,30 +154,27 @@ instance BlobStore BlobDirStore where
   getBlob bd (ExtantBlob name) = sourceFile (blobFileName bd name)
 
 blobFileName :: BlobDirStore -> BS.ByteString -> FilePath
-blobFileName (BlobDir d) blobname = d </> toString (Base64.encode blobname)
+blobFileName (BlobDir d) = (d </>) . toString . Base64.encode
```

Comment:
Definitely a nit, but I like to keep things point free unless it makes things confusing.